command/genesis/polybft_params.go

```diff
@@ -127,6 +136,7 @@ func (p *genesisParams) getGenesisValidators(validators []GenesisTarget) (result
 			result = append(result, &polybft.Validator{
 				Address: types.StringToAddress(parts[0]),
 				BlsKey:  parts[1],
+				Balance: big.NewInt(1000), // hard-coded balance
```

Comment:
Can we specify these hardcoded values in some const or at least var (if we want bigint)

Reply:
Realized that was not correct way to set it and it is completely changed now. Since we have such information written in the genesis (and is dictated by genesis cmd premine flag), I've used that value to initialize validators balance).
@igorcrevar please take a look whether it makes sense to you:
[dad6eba](https://github.com/0xPolygon/polygon-edge/pull/796/commits/dad6ebaa45aa22820d59d8f3f3efde388220bbc9)